src/yunohost/tools.py

```diff
@@ -127,6 +129,9 @@ def tools_adminpw(auth, new_password):
 
     """
     from yunohost.user import _hash_user_password
+    from yunohost.utils.password import LoggerPasswordValidator
+
+    LoggerPasswordValidator('admin').validate(new_password)
```

Comment:
Those two lines are repeated at several places ... I'd rather have a call to `tools_validatepw` (or something like `_assert_password_is_strong_enough`)